python/lsst/sims/utils/timeTransformations.py

```diff
@@ -0,0 +1,195 @@
+"""
+For definitions of all (or, at least, most of) the time systems handled in this file, see
+
+https://www.cv.nrao.edu/~rfisher/Ephemerides/times.html
+"""
+
+import numpy as np
+import warnings
+import palpy
+from lsst.sims.utils import Ut1MinusUtcData
+from lsst.sims.utils import TT_from_TAI_Data
+
+__all__ = ["taiFromUtc", "utcFromTai",
+           "dutFromUtc",
+           "ut1FromUtc", "utcFromUt1",
+           "dttFromUtc", "ttFromTai",
+           "tdbFromTt"]
+
+
+def taiFromUtc(utc):
+    """
+    Use the palpy method palDat to convert from Coordinated Universal Time
+    (UTC) to International Atomic Time (TAI)
+
+    @param [in] utc is the UTC time as an MJD
+
+    @param [out] TAI time as an MJD
+    """
+
+    dt = palpy.dat(utc) # returns TAI-UTC in seconds
+    return utc + dt/86400.0
```

Comment:
86400 is used throughout this set of functions. Why not make it a labelled variable with global scope within the file? (i.e. just put it at the head and give it a descriptive name).